doc/development/stack.md

```diff
@@ -0,0 +1,95 @@
+# Stack of puter
+
+Puter source is modular, meaning that there are a **node module for each src/ folder subdirectory**.
+That means each module could be defining its own package dependency list.
+
+## General
+
+1. Using Javascript as main language.
+2. Using DHTML (html,js,css) for dinamic web rendering.
+3. Using Shell and Docker for building and deployment scripts.
+4. Using Docker for containerized deployment.
+5. Using SQLITE for persistence (of sessions, app storage...)
+6. Using local disk for some storage operations.
+
+## Node related
+
+Puter currently Node-related stack is:
+
+### DevDependencies
```

Comment:
If we list dependencies here we have to use a script to generate this, otherwise it adds too much maintenance overhead. We should be able to get package descriptions from npm if it's automated. I would suggest removing this from this PR (so we can get it merged more quickly), and scripting/automating this in a subsequent PR.